client/extensions/woocommerce/state/products/test/actions.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { WOOCOMMERCE_EDIT_PRODUCT } from '../../action-types';
+import { editProduct } from '../actions.js';
+
+describe( 'actions', () => {
+	it( 'should create an action to edit a product', () => {
```

Comment:
I wonder how high value of a test this is. It's a fairly simple operation to construct an action, maybe it doesn't need a separate test just for that.

This could probably be tested just as effectively by passing the action into the reducer and observing that the resulting state is as desired.